### PR TITLE
Add example of using reduce method on array of objects

### PR DESCRIPTION
**Pull request by oliviap103**
- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/arrays/terms/reduce/reduce.md [hunk 1]
```diff
@@ -50,13 +50,22 @@ console.log(
   'Adding with initial value:',
   arrayTwo.reduce((acc, curr) => acc + curr, 'a')
 );
+
+// Add the values of each object inside an array.
+const arrayThree = [{x:1}, {x:2}, {x:4}];
+
+console.log(
+  'Adding object values:',
+  arrayThree.reduce((arr, curr) => acc + curr.x, 0)
```
Comment (SSwiniarski) [inline content/javascript/concepts/arrays/terms/reduce/reduce.md:59]: should this be "(acc, curr)" ?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@oliviap103 I did a review and left a comment.

File: content/javascript/concepts/arrays/terms/reduce/reduce.md (hunk 1 above)
Comment (oliviap103) [reply content/javascript/concepts/arrays/terms/reduce/reduce.md]: @SSwiniarski No problem thank you for the clarification - that should be it changed now.

**Review by oliviap103 — COMMENTED** [review]
changed arr to acc

**Review by SSwiniarski — APPROVED** [review]
@oliviap103 looks good for a second review

**Review by yangc95 — APPROVED** [review]
LGTM

**Comment by oliviap103** [comment]
@yangc95 Hi, Is there anything else I still have to do? Sorry, I am new to this!

**Comment by oliviap103** [comment]
I'm a bit confused sorry - I'm happy for someone else to take over the issue.

**Comment by SSwiniarski** [comment]
@oliviap103, sorry for the delay. This should have been merged a while ago.

**Pull request merged**